horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
Even more so, using more than two spaces as well. So what's the reasoning behind using two spaces then?

Before I wrote the comment, I did a simple check across the codebase. We have far more inline comments with one space than two. So in my opinion, that's the way to go.